packages/main/test/samples/Tree.sample.html

```diff
@@ -0,0 +1,188 @@
+<header>
```

Comment:
To display since there is special marker:


```html
<header class="component-heading">
	<h2 class="control-header">Tree</h2>
	<div class="component-heading-since">
		<span><!--since_tag_marker--></span>
	</div>
</header>
```